app/components/concept/index.hbs

```diff
@@ -1,4 +1,9 @@
-<div {{did-insert this.handleDidInsertContainer}} {{will-destroy this.handleWillDestroyContainer}} ...attributes>
+<div
+  {{did-insert this.handleDidInsertContainer}}
+  {{did-update this.recalculateLastRevealedBlockGroupIndex @latestConceptEngagement}}
```

Comment:
@VasylMarchuk shouldn't this just work by default? We're using `latestConceptEngagement.currentProgressPercentage` inside a function, as long as that is tracked, the value should update automatically no?